### PR TITLE
Add support for Livisi remotes and wall buttons

### PR DESCRIPTION
**Pull request by planbnet**
## Proposed change

Document support for Livisi wall buttons similar to the [Hue integration docs](https://www.home-assistant.io/integrations/hue/#hue-remotes-and-switches)

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/89258
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/livisi.markdown [hunk 1]
```diff
@@ -40,3 +41,7 @@ The current integration will not find your SHC automatically and needs to be con
 ## Device Discovery
 
 All devices are automatically discovered and included by the integration. If you include a new device in LIVISI SmartHome, the device will automatically appear in Home Assistant after a few minutes.
+
+## Remotes and wall buttons
+
+Livisi remotes such as the 8 Button remote (BRC8) and wall buttons (WSC2) are stateless devices, meaning that they do not have a on/off state like regular entities in Home Assistant. Instead, such devices emit the event `livisi_event` when a button is pressed. You can test what events come in using the event {% my developer_events title="developer tools in Home Assistant" %} and subscribe to the `livisi_event`. Once you know what the event data looks like, you can use this to create automations.
```
Comment (frenck) [inline source/_integrations/livisi.markdown:47]: We should implement device triggers instead. Those will provide a UI and are more user-friendly. We should not be sending users into developer tools.

<https://developers.home-assistant.io/docs/device_automation_trigger>

Comment (planbnet) [reply source/_integrations/livisi.markdown:47]: I read about device triggers and then wanted to look for an example in one of the other integrations. That's when I found this snippet in the Hue integration docs: https://www.home-assistant.io/integrations/hue/#hue-remotes-and-switches. As Hue is one of the very mature integrations, I figured that device triggers cannot be used when there are no entities. 
Do you happen to know an example where device triggers are used to model stateless hardware buttons?

Comment (frenck) [reply source/_integrations/livisi.markdown:47]: > As Hue is one of the very mature integrations

Hue is mostly old and has history, not all integration implement all modern things yet.

> I figured that device triggers cannot be used when there are no entities.

Nope, device triggers are not limited to entities at all.

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

The docs look good and are matching the PR right now, so approved it for now. I think the comment above still holds true, but let's see what the parent PR review will bring up.

**Pull request closed without merging**